vcf/filter.go

```diff
@@ -201,11 +58,23 @@ func IsSegregating(v Vcf) bool {
 	if len(v.Samples) == 0 {
 		return false //special case, no samples
 	}
-	var firstEncountered int16 = v.Samples[0].AlleleOne
+
+	var firstEncounteredBase int16
+	var firstEncountered bool = false
 
 	for _, sample := range v.Samples {
-		if sample.AlleleOne != firstEncountered || sample.AlleleTwo != firstEncountered {
-			return true
+		if len(sample.Alleles) == 0 {
```

Comment:
Can this happen?

Reply:
Yes, if the genotype field is not present, but other format fields are.